app/views/layouts/application.html.erb

```diff
@@ -5,9 +5,10 @@
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
+  <script src="http://d3js.org/d3.v3.min.js"></script>
```

Comment:
If the map only appears on the home page, we wouldn't want this script to be loaded on all the other pages as well.  Could you update this so it's not loaded for every page?